### PR TITLE
samples: move AutoML vision samples from nodejs-vision

### PR DESCRIPTION
**Pull request by jkwlui**

### CODE REVIEW

File: samples/system-test/automlVision.test.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+const cmdPredict = `node automl/automlVisionPredict.js`;
+
+const testDataSetName = `testDataSet`;
+const dummyDataSet = `dummyDataSet`;
+const testModelName = `dummyModel`;
+const testImgPath = `./resources/`;
+const sampleImage2 = path.join(testImgPath, `testImage2.jpg`);
+
+describe(`auto ml vision`, () => {
+  it.skip(`should create, list, and delete a dataset`, async () => {
```
Comment (callmehiphop) [inline samples/system-test/automlVision.test.js]: Are these all skipped intentionally?

Comment (beccasaurus) [reply samples/system-test/automlVision.test.js]: @nirupa-kumar @sirtorry are these all long running, e.g. which require model training? Or can any of these actions be performed rather instantaneously?

Comment (nirupa-kumar) [reply samples/system-test/automlVision.test.js]: All of them took longer to complete and the time duration had not been consistent. Some of them do require Model training. That was the reason we moved to skipping them. Please let me know if there is anything else we could do.

Comment (sirtorry) [reply samples/system-test/automlVision.test.js]: let me rethink the tests

**Pull request merged**